engine/src/main/java/org/terasology/rendering/dag/WireframeStateChange.java

```diff
@@ -15,11 +15,12 @@
  */
 package org.terasology.rendering.dag;
 
+
 /**
  *
  */
-public class WireframeStateChange extends AbstractStateChange<Boolean> {
-    WireframeStateChange(Boolean value) {
+public class WireframeStateChange extends AbstractStateChange<Integer> {
```

Comment:
I really wish you had switched to a different state change type, i.e. the FBO binding operation like a I suggested. 

This wireframe thing is really not that important at this stage and especially is not particularly representative of the state changes we'll be mostly dealing with. 

Consider that the underlying opengl commands for FBO binding are really simple and are integer-based. Even the display is just another integer id.